Sources/SentryCrash/Recording/Tools/SentryCrashSysCtl.c

```diff
@@ -233,7 +233,7 @@ bool
 sentrycrashsysctl_getMacAddress(const char *const name, char *const macAddressBuffer)
 {
     // Based off
-    // http://iphonedevelopertips.com/device/determine-mac-address.html
+    // http://web.archive.org/web/20111109013928/http://iphonedevelopertips.com/device/determine-mac-address.html
```

Comment:
The original link is broken now, but I found a version in the web archive. 
Also not sure if should keep this around.